notebooks/concepts/Fitting_multiprocess.ipynb

```diff
@@ -0,0 +1,1742 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Read spectrum data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is a 74 X 74 spatial image in which each spaxel represents a spectrum with 4563 data points."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(4563,)\n",
+      "(4563, 74, 74)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# fn = download_file('https://data.sdss.org/sas/dr14/manga/spectro/redux/v2_1_2/7495/stack/manga-7495-12704-LOGCUBE.fits.gz', cache=True)\n",
+    "fn = download_file('file:/Users/busko/Desktop/manga-7495-12704-LOGCUBE.fits', cache=True)\n",
+    "f = fits.open(fn)\n",
+    "\n",
+    "flux3d = f[1].data\n",
+    "wave = f[4].data\n",
+    "\n",
+    "print(wave.shape)\n",
+    "print(flux3d.shape)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Units handling is presumed to come from above, perhaps at the loader level?\n",
+    "# Here we pretend we got them already somehow\n",
+    "funit_string = f[1].header['BUNIT']\n",
+    "funit = u.def_unit(funit_string, 1.E-17 * u.erg/u.s/u.cm/u.cm/u.Angstrom)\n",
+    "u.add_enabled_units([funit])\n",
+    "wunit = u.Unit('Angstrom')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define central spaxel, and area around it"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can change the area to a smaller cube to speed up the notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CENTER = 37   # center of square spaxel region\n",
+    "HSIZE = 15    # half-size of square"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Generate list of all spaxels to be fitted\n",
+    "spx = [[(x,y) for x in range(CENTER-HSIZE, CENTER+HSIZE)] for y in range(CENTER-HSIZE, CENTER+HSIZE)]\n",
+    "SPAXELS = [item for sublist in spx for item in sublist]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fit central spaxel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Fit the central spaxel to use the fit results as initial values for the subsequent fits over other spaxels. \n",
+    "\n",
+    "In here, we use the code from `Example_fitting.ipynb`. We disregard the error array for now, since the purpose is not to measure the input data itself, but instead to measure performance."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define spectrum at the central spaxel"
+   ]
```

Comment:
This is the right idea--fit a representative spectrum to get starting values for the fits of all spaxels. The viz tool should have the flexibility to do this either on a user-specified spaxel or another spectrum, for example the average spectrum of the cube.  The advantage of the latter is that not all spaxels will contain all spectral features of interest, and also that the user doesn't have to think about which spaxel is best.